vico/views/src/main/java/com/patrykandpatrick/vico/views/chart/BaseChartView.kt

```diff
@@ -500,7 +500,13 @@ public abstract class BaseChartView<Model : ChartEntryModel> internal constructo
         }
     }
 
+    @Suppress("SENSELESS_COMPARISON")
     override fun onRtlPropertiesChanged(layoutDirection: Int) {
-        measureContext.isLtr = layoutDirection == ViewCompat.LAYOUT_DIRECTION_LTR
+        // This function may be invoked inside of the View’s constructor, before the measureContext is initialized.
+        // In this case, we can ignore this callback, as the layout direction will be determined when the MeasureContext
+        // instance is created.
+        if (measureContext != null) {
+            measureContext.isLtr = layoutDirection == ViewCompat.LAYOUT_DIRECTION_LTR
+        }
```

Comment:
I’d suggest using a guard clause here.

```suggestion
        if (measureContext == null) return
        measureContext.isLtr = layoutDirection == ViewCompat.LAYOUT_DIRECTION_LTR
```